### DISCUSSION TITLE
Being able to assign a path for the different fbclient.dll versions

### DISCUSSION THREAD

**Discussion by pepegaucho** [mariuz/flamerobin#394] [Ideas]
Maybe this possibility already exists. My idea is to have in the "Preferences" a list of paths for the different versions of fbclient.dll. If it isn't indicated, take the default path. For example, a grid or string like key-value with version-path: "3.0 -> c:\firebird\firebird_3_0\; 4.0 -> c:\firebird\firebird_4_0\". This would help when I have several versions of FB installed on the same OS

**Comment by Wolf-SO** [comment]
@pepegaucho In case you don't know this already: it is possible to work with different servers via TCP/IP connections. If the servers run on the same machine, they have to use different ports. Ports different than 3050 (which is the default), have to be specified in the connect string like this:

```
server/port:databasepath
```
For more, see: [Which TCP/IP port is Firebird using?](https://firebirdfaq.org/faq259/ "Which TCP/IP port is Firebird using?") and [Compatibility of Firebird client with Firebird server](https://stackoverflow.com/questions/19049645/compatibility-of-firebird-client-with-firebird-server/19141169#19141169 "Compatibility of Firebird client with Firebird server - Stack Overflow")
